stubs/pywin32/win32com/server/dispatcher.pyi

```diff
@@ -17,6 +17,12 @@ class DispatcherWin32trace(DispatcherTrace):
 class DispatcherOutputDebugString(DispatcherTrace): ...
 
 class DispatcherWin32dbg(DispatcherBase):
+    @deprecated(
```

Comment:
Should this be on the class instead?

Reply:
Oh yeah you can do that! Any usage of the class (even as type hint) appears as deprecated in Pylance. Sure.